hipflowapp/Hip_Native/Data/insert_Extract.cpp

```diff
@@ -237,7 +238,7 @@ uint8_t c_insert_Extract::extract( float   & x, uint8_t **ppData, uint8_t& dataC
 	//uint16_t g = (uint16_t)((x >> 16) & 0xffff);// hi word
```

Comment:
Needs a explanation.

Reply:
remove the code except the last line, which needed to be retained.  i tested it.